schema.go

```diff
@@ -75,6 +75,35 @@ const (
 	Duration        LogicalType = "duration"
 )
 
+func isNative(typ Type) bool {
+	switch typ {
+	case Null, Boolean, Int, Long, Float, Double, Bytes, String:
+		return true
+	default:
+	}
+
+	return false
+}
+
+func isPromotable(typ Type) bool {
+	switch typ {
+	case Int, Long, Float, String, Bytes:
+		return true
+	default:
+	}
+
+	return false
+}
+
+// Action is a field action used during decoding process.
+type Action string
+
+// Action type constants.
+const (
+	FieldDrain      Action = "drain"
```

Comment:
What do you think of renaming this to `FieldIgnore`? Drain is no super obvious to me as to what it will do.

Reply:
`FieldIgnore` sounds good too. I think I borrowed "Drain" from the official Java implementation.

I'll change `FieldDrain`by `FieldIgnore`.